odl/contrib/pyshearlab/examples/wave_shear_separation.py

```diff
@@ -0,0 +1,53 @@
+import odl
+import numpy as np
+from odl.contrib.pyshearlab import PyShearlabOperator
+import scipy.misc
+
+img = scipy.misc.imread(
+   '/home/hkohr/Downloads/Yv2Ki.png')
```

Comment:
Image path should not be hard coded. Are there example images to be shipped with ODL? This may be good, but GIT is probably not the right way how to do it.

Reply:
Totally agree with @MatthiasJE here. We should simply replace this with something standard (or if we REALLY like the image, we can move it to `odl.contrib.datasets` and load it lazily).

Reply:
Good point, I overlooked that one. We could take something that we can simply pull from the web, like from Wikipedia. I think the image I used there wasn't really useful anyway.

Reply:
We could consider using some of the images in the paper "Image Separation Using Wavelets and Shearlets" (http://link.springer.com/chapter/10.1007/978-3-642-27413-8_26), in particular those in figure 1 and 6. I guess we may have to ask Gitta Kutyniok for permission.

Reply:
> in particular those in figure 1 and 6. I guess we may have to ask Gitta Kutyniok for permission.

That and we need a direct link, otherwise it's hard to use. But yes, these images would be ideal.

Reply:
The images are hosted [here](http://www3.math.tu-berlin.de/numerik/www.shearlab.org/applications) somewhere, so there's a nonzero chance of getting a permalink.

Reply:
A link sounds shaky, can't we ask for permission to include them in odl.contrib.datasets?

Reply:
odl.contrib.datasets does not host anything on its own, since we have nowhere to store it :D

Reply:
Well, a rather blunt way to do this would be to make a new GitHub repository only for (relatively small) datasets and use the raw link to the files.

Reply:
There's also [Git LFS](https://git-lfs.github.com/) (large file support) but it requires a Git addon.

Reply:
Frankly I've been trying to salvage this but the example is very hard to get to work "easily" so to say. It really needs a longer explanation. My suggestion is we remove it

Reply:
Sure, I don't mind. But the larger issue with file hosting stands. Can't we make a repo for that and use raw links for downloading in `odl.contrib.datasets`?